C/main_libFuzzer_extra_counters.c

```diff
@@ -0,0 +1,9 @@
+// Copyright 2015 go-fuzz project authors. All rights reserved.
+// Use of this source code is governed by Apache 2 LICENSE that can be found in the LICENSE file.
+
+#ifdef __linux__
+__attribute__((section("__libfuzzer_extra_counters")))
+#else
+#error Currently only Linux is supported
+#endif
+unsigned char GoFuzzCoverageCounters[65536];
```

Comment:
I still fail to see how this file is involved in the build. It doesn't seem to be.
I think we should get rid of this file entirely now and instead do this right in the go-fuzz-build/main.go main package text as:
```
// #cgo CFLAGS: -Wall -Werror
/*
#ifdef __linux__
__attribute__((weak, section("__libfuzzer_extra_counters")))
#else
#error Currently only Linux is supported
#endif
unsigned char GoFuzzCoverageCounters[65536];
*/
import "C"
```

This makes things simpler and more self-contained and produces working fuzzer with just:
````
clang fmt.a -fsanitize=fuzzer
```

Reply:
Fixed, it is now completely free of C files.

Reply:
+1